src/utils/logging.py

```
@@ -1,558 +1,310 @@
-    lines = ["# ⚖️ LLM Judge Input\n"]
+    Returns:
+        The decorated function if Langfuse is configured, otherwise the original function
+    """
+    if settings.LANGFUSE_PUBLIC_KEY:
+        # Import here to avoid circular imports and only import when needed
+        from langfuse.decorators import observe  # pyright: ignore
 
-    lines.append("### 🎯 Configuration")
-    lines.append(f"**Max Retries:** {max_retries}")
+        return observe()(func)
+    else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)